src/components/modals/export-map-modal.js

```diff
@@ -58,41 +57,198 @@ const StyledInput = styled.input`
   }
 `;
 
-const propTypes = {
-  exportHtml: PropTypes.object,
-  onExportMapboxAccessToken: PropTypes.func.isRequired
+const StyledWarning = styled.span`
+  color: ${props => props.theme.errorColor};
+  font-weight: ${props => props.theme.selectFontWeightBold};
+`;
+
+const StyledLink = styled.a`
+  text-decoration-line: underline !important;
+`;
+
+const INTRA_SECTION_MARGING = '8px';
+
+const ExportMapStyledExportSection = styled(StyledExportSection)`
+  .disclaimer {
+    font-size: ${props => props.theme.inputFontSize};
+    color: ${props => props.theme.inputColor};
+    margin-top: 12px;
+  }
+`;
+
+const Link = ({children, ...props}) => (
+  <StyledLink target="_blank"
+              rel="noopener noreferrer" {...props}>
+    {children}
+  </StyledLink>
+);
+
+const exportHtmlPropTypes = {
+  options: PropTypes.object,
+  onEditUserMapboxAccessToken: PropTypes.func.isRequired
 };
 
-const ExportMapModal = ({
-  exportHtml,
-  onExportMapboxAccessToken
+const ExportHtmlMap = ({
+  options = {},
+  onEditUserMapboxAccessToken = () => {}
 }) => (
-  <StyledExportMapModal>
-    <div className="export-map-modal">
-      <div className="title">
-        Export your map into a single html file
+  <div>
+    <StyledExportSection style={{marginTop: INTRA_SECTION_MARGING}}>
+      <div className="description" />
+      <div className="selection">
+        Export your map into an interactive html file.
       </div>
-      <div className="subtitle">
-        You will be able to save your map into an interactive html file.
-      </div>
-
-      <StyledSection>
-        <div>
-          Please provide your <a style={{textDecorationLine: 'underline'}} href={MAPBOX_ACCESS_TOKEN}>Mapbox access token</a>
+    </StyledExportSection>
+    <ExportMapStyledExportSection className="export-map-modal__html-options">
+      <div className="description">
+        <div className="title">
+          Please provide your Mapbox access token
         </div>
+        <div className="subtitle">
+          Export your map into a single html file
```

Comment:
put `Use your own Mapbox access token in the html (optional)`

Reply:
done